official-services.md

```diff
@@ -11,10 +10,7 @@ The following services are officially operated by the FEniCS Project.
 - [PyPi](https://pypi.org/user/fenicsproject/)
 - [Slack](https://fenicsproject.slack.com)
 - [Steering Council mailbox](mailto:fenics-steering-council@googlegroups.com)
-- [Support mailing list](https://groups.google.com/forum/#!forum/fenics-support)
-- [Twitter feed curated from #fenicsnews tag](https://twitter.com/search?q=%23fenicsnews&src=typd)
 - [Ubuntu PPA](https://launchpad.net/~fenics-packages/+archive/ubuntu/fenics)
-- [UFL readthedocs.org](https://fenics.readthedocs.io/projects/ufl/en/latest/)
```

Comment:
The new versions seem to be pushed on `docs.fenicsproject.org`, while `fenics.readthedocs.io` is still at `2019.x.y`.